src/components/Article/CommentInput.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import agent from '../../agent';
 import { connect } from 'react-redux';
-import { ADD_COMMENT } from '../../constants/actionTypes';
+
+import { addComment } from '../../reducers/article';
 
 const mapDispatchToProps = dispatch => ({
```

Comment:
:hand: Some nitpicks here:

- I don't see a reason to leave any of these as class components - all components should really be converted to functions
- If we were actually going to keep  `connect`, this should be using the "object shorthand" form of `mapDispatch`, but I'd also like to see them switched over to use the hooks API instead of `connect`

Other than that, I like the direction of this PR - keep it up!

Reply:
Yes, of course, I'm converting all the reducers to slices then I will start converting class component to functional components and replace connect over hooks, and when everything is working I will move things to follow the features structure

Reply:
sounds great! thank you for working on this!